src/layout.jl

```diff
@@ -20,20 +20,20 @@ struct Vec{N, T} end
 
 @inline @generated function vloada(::Type{Vec{N, T}}, ptr::Core.LLVMPtr{T, AS}, i::Integer = 1) where {N, T, AS}
     alignment = sizeof(T) * N
-    vec_len = (sizeof(T) * N) ÷ sizeof(Float32)
+    vec_len = (sizeof(T) * N) ÷ sizeof(T)
```

Comment:
This can be removed and replaced by just `N` now.

Reply:
:facepalm: